compiled_starters/java/src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+// Uncomment this block to pass the first stage
+//  import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        // You can use print statements as follows for debugging, they'll be visible when running tests.
+        System.out.println("Logs from your program will appear here!");
+
+        System.out.print("$ ");
```

Comment:
Let's include the prompt inside the commented bit like other challenges do: 

<img width="768" alt="Screenshot 2024-06-05 at 11 56 04" src="https://github.com/codecrafters-io/build-your-own-shell/assets/3893573/25dc83b5-2bff-4a1b-9c7c-dd9a874b9733">